docs/csharp/language-reference/keywords/verbatim.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "@ (C# Reference) | Microsoft Docs"
+ms.date: "2017-02-09"
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+f1_keywords: 
+  - "@_CSharpKeyword"
+  - "@"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - @ keyword [C#]"
+ms.assetid: 0653146b-632b-423d-9e33-affd213b7ee5
+author: "rpetrusha"
+ms.author: "ronpet"
+translation.priority.ht: 
+  - "cs-cz"
+  - "de-de"
+  - "es-es"
+  - "fr-fr"
+  - "it-it"
+  - "ja-jp"
+  - "ko-kr"
+  - "pl-pl"
+  - "pt-br"
+  - "ru-ru"
+  - "tr-tr"
+  - "zh-cn"
+  - "zh-tw"
+---
+# @ (C# Reference)
+
+The `@` character serves as a verbatim identifier. It can be used in the following ways:
+
+1. To enable C# keywords to be used as identifiers. The `@` character prefixes a code element that the compiler is to interpret as an identifier rather than a C# keyword. The following example uses the `@` character to define an identifier named `for` that it uses in a `for` loop.
+
+   [!code-csharp[verbatim1](../../../../samples/snippets/csharp/language-reference/keywords/verbatim1.cs#1)]
```

Comment:
I'm not sure why. Maybe it's a side-effect of the missing quotation mark previously. I've added a space before [verbatim], though I doubt it's significant. The path is correct, and the snippet tag is present.